lib/l10n/app_en.arb

```diff
@@ -0,0 +1,42 @@
+{
+    "nameWallet": "Wallet",
+    "@nameWallet": {
+      "description": "title of the page wallet"
+    },
```

Comment:
nao precisa desses @ so usa ele quando for importante

Reply:
ajuste realizado 
![image](https://user-images.githubusercontent.com/103940293/181344151-e936216a-42ee-47ba-a1d0-4d613c37e9e8.png)